### PR TITLE
PE-3988: Adds new Pin icon

### PR DESCRIPTION
**Pull request by matibat**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/36es4okk50b80

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -87,11 +87,11 @@ packages:
...
     description:
       path: "."
-      ref: PE-4295-update-label-color-of-ar-drive-text-field
-      resolved-ref: "8bbb34d41f60b990018c933ae1eea8e5189724a7"
+      ref: PE-3988
+      resolved-ref: "9175bd2b97bb5ee3cf861cd140e5e3d7944e460a"
       url: "https://github.com/ar-io/ardrive_ui.git"
     source: git
-    version: "1.5.1"
+    version: "1.5.0"
```
Comment (thiagocarvalhodev) [inline pubspec.lock:94]: @matibat , please merge the latest version from `ardrive_ui` on your branch

**Review by thiagocarvalhodev — CHANGES_REQUESTED** [review]
Merge the latest version from `ardrive_ui` on your ardrive_ui branch

File: pubspec.lock (hunk 1 above)
Comment (matibat) [reply pubspec.lock]: There aa4eb63f4b1cda69a2c03765a22abf23e67963a0

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM!

**Pull request merged**